SU2_CFD/src/fluid/CFluidScalar.cpp

```diff
@@ -212,13 +214,43 @@ su2double CFluidScalar::ComputeMeanSpecificHeatCp(const su2double* val_scalars)
   return mean_cp;
 }
 
+su2double CFluidScalar::ComputeEnthalpyFromT(const su2double val_temperature, const su2double* val_scalars){
+  /* In future implementations, enthalpy should be computed using numerical integration. For now, as Cp does not
```

Comment:
You should try to come up with a way of directly computing enthalpy instead of computing an integral everytime this is called.